app/assets/javascripts/discourse/app/templates/invites/show.hbs

```diff
@@ -39,6 +40,15 @@
           {{/if}}
         {{/if}}
 
+        {{#if discourseConnectEnabled}}
+          <div id="login-buttons">
+            <button type="button" class="btn btn-social discourse-connect" {{on "click" (action "ssoLogin")}}>
```

Comment:
This feels weird, is there a reason we can't use a `a` tag here?

Reply:
Good point probably not, I think I had it as a button because at first `ssoLogin` was doing more than a redirect